### ISSUE TITLE
Event für geänderte Systemvariablen

### ISSUE THREAD

**Issue by McLive** [jens-maus/RaspberryMatic#258]
Wie man [hier](https://docs.openhab.org/addons/bindings/homematic/readme.html#troubleshooting) lesen kann, sendet die CCU nur ein Event, wenn sich ein Datenpunkt eines Gerätes ändert.
Schön wäre es, wenn dies auch mit Systemvariablen funktioniert, damit man diese ohne Workarounds auch in openHAB darstellen kann.

``
Variables out of sync

The CCU only sends a event if a datapoint of a device has changed. There is (currently) no way to receive a event automatically when a variable has changed. To reload all variable values, send a REFRESH command to any variable.
e.g you have a item linked to a variable with the name Var_1
In the console:
``

**Comment by jp112sdl** [comment]
Das wird sich m.M. nach nicht umsetzen lassen.

In ioBroker sieht es ähnlich aus. Auch hier werden Variablen nur in Intervallen abgefragt.
Es gibt jedoch einen Workaround, diesen Poll anzustoßen, in dem man den virtuellen Taster 50 betätigt.
Vielleicht hilft dir der [Beitrag im ioBroker Forum](forum.iobroker.net/viewtopic.php?t=4084) weiter.

**Comment by robdg80** [comment]
Der Wunsch wäre somit, das man Rega Systemvariablen auch mittels xml-rpc "bereitstellt", damit diese auch eventbasiert abgefragt werden können.

**Comment by jens-maus** [comment]
Darüber könnte man in der Tat nachdenken, das würde aber schon ziemlich arbeitet bedeuten das alles umzusetzen in der ReGa. Werde es aber auf die Wunschliste setzen. Bis dahin (das kann sicher einige zeit dauern – gibt wichtigeres) bleibt einem nichts weiter übrig weiterhin ein polling durchzuführen wie ioBroker das tut.

**Comment by McLive** [comment]
Genau! Aktuell nutze ich auch den Workaround mit der virtuellen Taste, um entsprechende Variablen dann zu aktualisieren. 

Eine eventbasierte Lösung würde ich jedoch deutlich besser finden. Für die Geräte funktioniert es ja immerhin auch schon. :)

**Comment by jp112sdl** [comment]
Klingt gut! Darüber freut sich dann auch die ioBroker Community.
